OpenObserve - Updating OpenObservecurl: (22) The requested URL returned error: 404

### ✅ Have you read and understood the above guidelines?

yes

### 📜 What is the name of the script you are using?

OpenObserve

### 📂 What was the exact command used to execute the script?

update and bash -c "$(curl -fsSL https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/ct/openobserve.sh)"

### ⚙️ What settings are you using?

- [ ] Default Settings
- [x] Advanced Settings

### 🖥️ Which Linux distribution are you using?

Debian 12

### 📝 Provide a clear and concise description of the issue.

An update is not possible as the file cannot be found because the variable {LATEST} in the URL https://github.com/openobserve/openobserve/releases/download/$LATEST/openobserve-${LATEST}-linux-amd64.tar.gz is not replaced.
 

### 🔄 Steps to reproduce the issue.

1. open lxc Container
2. update 

or: 
1. open lxc Container
2. bash -c "$(curl -fsSL https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/ct/openobserve.sh)"

### ❌ Paste the full error output (if available).

   ____                   ____  __                            
  / __ \____  ___  ____  / __ \/ /_  ________  ______   _____ 
 / / / / __ \/ _ \/ __ \/ / / / __ \/ ___/ _ \/ ___/ | / / _ \
/ /_/ / /_/ /  __/ / / / /_/ / /_/ (__  )  __/ /   | |/ /  __/
\____/ .___/\___/_/ /_/\____/_.___/____/\___/_/    |___/\___/ 
    /_/                                                       

  ⏳   Updating OpenObservecurl: (22) The requested URL returned error: 404

gzip: stdin: not in gzip format
tar: Child returned status 1
tar: Error is not recoverable: exiting now

[ERROR] in line 33: exit code 0: while executing command tar zxvf <(curl -fsSL https://github.com/openobserve/openobserve/releases/download/$LATEST/openobserve-${LATEST}-linux-amd64.tar.gz) -C /opt/openobserve

### 🖼️ Additional context (optional).

_No response_

--- a/ct/openobserve.sh
+++ b/ct/openobserve.sh
@@ -30,7 +30,7 @@ function update_script() {
     msg_info "Updating $APP"
     systemctl stop openobserve
     LATEST=$(curl -fsSL https://api.github.com/repos/openobserve/openobserve/releases/latest | grep '"tag_name":' | cut -d'"' -f4)
-    tar zxvf <(curl -fsSL https://github.com/openobserve/openobserve/releases/download/$LATEST/openobserve-${LATEST}-linux-amd64.tar.gz) -C /opt/openobserve
+    $STD tar zxvf <(curl -fsSL https://downloads.openobserve.ai/releases/openobserve/$LATEST/openobserve-$LATEST-linux-amd64.tar.gz) -C /opt/openobserve
     systemctl start openobserve
     msg_ok "Updated $APP"
     exit

--- a/frontend/public/json/openobserve.json
+++ b/frontend/public/json/openobserve.json
@@ -9,7 +9,7 @@
   "updateable": true,
   "privileged": false,
   "interface_port": 5080,
-  "documentation": null,
+  "documentation": "https://openobserve.ai/docs/",
   "website": "https://openobserve.ai/",
   "logo": "https://cdn.jsdelivr.net/gh/selfhst/icons/webp/openobserve.webp",
   "config_path": "/opt/openobserve/data/.env",

